webrtc-stats.html

```diff
@@ -1298,6 +1299,17 @@ <h2>
                   the first frame is decoded this attribute is missing.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>frameBitDepth</code></dfn> of type <span class="idlMemberType">unsigned
+                long</span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for video. Represents bit depth per channel of the raw video frame.
+                  Typical values are 8, 10, or 12 bits.
+                  Before the first frame is decoded this attribute is missing.
```

Comment:
What's the benefit of "per channel" versus "per pixel"? Maybe same same but different but it makes me wonder, what if the format is no traditional 3 channel of same depth

Reply:
Yes, that's probably better. I've just pushed a change to per pixel.